squidDrone/tests/mock_libraries/com/com_message_buffer_mock.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef TESTS_MOCK_LIBRARIES_COM_COM_MESSAGE_BUFFER_HPP_
+#define TESTS_MOCK_LIBRARIES_COM_COM_MESSAGE_BUFFER_HPP_
+
+#include <queue>
+#include <array>
+
+namespace com 
+{
+
+  class ComMessageBuffer {
+    public:
+      ComMessageBuffer(){
+        _data.fill(0xaa);
+        test_member = 0;
```

Comment:
please add test_member either to the class member initializer list of the constructor or do a default initialization.